ComponentKit/Core/Scope/CKComponentScopeHandle.mm

```diff
@@ -46,7 +41,7 @@ + (CKComponentScopeHandle *)handleForComponent:(CKComponent *)component
     }
     return handle;
   }
-  CKCAssertNil(controllerClassForComponentClass([component class]), @"%@ has a controller but no scope! "
+  CKCAssertNil(CKComponentControllerClassFromComponentClass([component class]), @"%@ has a controller but no scope! "
```

Comment:
Because we call it now in more than one context. We use it to verify the action is either implemented on the component or its controller.